src/views/CurrentReservations.vue

```diff
@@ -1,59 +1,118 @@
 <template>
-<div>
-  <h3>Current Reservations</h3>
-  <b-container class="bv-example-row">
-  <b-row v-for="street in reservedStreets" :key="street.name" class="street-row">
-    <b-col><h6 class="reservation-item">
-      {{street.name + ' ' + street.type}}
-      </h6></b-col>
-    <b-button class="reservation-item"
-    @click="completeStreet(street.name + ' ' + street.type)" variant="success">
-      Complete</b-button>
-    <b-button class="reservation-item"
-    @click="releaseStreet(street.name + ' ' + street.type)" variant="danger">
-      Release</b-button>
-    <b-button class="reservation-item" @click="() => toEditReservations(street.FID)">
-      Edit reservation</b-button>
-  </b-row>
-  </b-container>
-</div>
+  <div>
+    <h1>Current Reservations</h1>
+    <p
+    v-if="currentReservations.length > 0"
+    class="basicText">
+      Press the ellipsis to complete, release or view your reservation
```

Comment:
Do we need this line? Is that in the designs? I think it'd be intuitive enough without.